sections/errorhandling/operationalvsprogrammererror.spanish.md

```diff
@@ -38,21 +72,21 @@ De la documentación oficial de Node.js
 
  > …By the very nature of how throw works in JavaScript, there is almost never any way to safely “pick up where you left off”, without leaking references, or creating some other sort of undefined brittle state. The safest way to respond to a thrown error is to shut down the process. Of course, in a normal web server, you might have many connections open, and it is not reasonable to abruptly shut those down because an error was triggered by someone else. The better approach is to send an error response to the request that triggered the error while letting the others finish in their normal time, and stop listening for new requests in that worker.
 
- > …Por la misma naturaleza de cómo throw funciona in JavaScript, casi nunca hay una forma segura de "seguir dónde lo dejaste", sin generar fugas de referencias, o sin crear algún otro tipo de estado frágil indefinido. La forma más segura de responder a un error lanzado is apagar el proceso. Claro, en un web server normal, podrías tener muchas conexiones abiertas, y no es rasonable cerrarlas abruptamente porque un error fue disparado por alguna otra persona. El mejor enfoque is enviar una respuesta de error a la petición que disparó el error mientras dejas las otras terminar en su tiempo normal, y parar de escuchar nuevas peticiones en ese worker.
+ > …Por la misma naturaleza de cómo throw funciona in JavaScript, casi nunca hay una forma segura de "seguir dónde lo dejaste", sin generar fugas de referencias, o sin crear algún otro tipo de estado frágil indefinido. La forma más segura de responder a un error lanzado es apagar el proceso. Claro, en un servidor web normal, podrías tener muchas conexiones abiertas, y no es razonable cerrarlas abruptamente porque un error fue disparado por alguna otra persona. El mejor enfoque es enviar una respuesta de error a la petición que disparó el error mientras dejas las otras terminar en su tiempo normal, y parar de escuchar nuevas peticiones en ese worker.
```

Comment:
untranslated:
   - `server -> servidor`
   - `is` -> `es`

typo:
   - `rasonable` -> `razonable`